src/SingleProject/Resizetizer/test/UnitTests/Resizetizer.UnitTests.csproj

```diff
@@ -9,14 +9,15 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.7.2" />
-    <PackageReference Include="Microsoft.Build.Framework" Version="16.7.0" PrivateAssets="all" />
-    <PackageReference Include="Microsoft.Build.Utilities.Core" Version="16.7.0" PrivateAssets="all" />
-    <PackageReference Include="SkiaSharp" />
-    <PackageReference Include="SkiaSharp.Extended" />
+    <PackageReference Include="Microsoft.Build.Framework" Version="17.7.2" PrivateAssets="all" />
+    <PackageReference Include="Microsoft.Build.Utilities.Core" Version="17.7.2" PrivateAssets="all" />
+    <PackageReference Include="SkiaSharp" Version="$(_SkiaSharpVersion)" />
+    <PackageReference Include="SkiaSharp.Extended" Version="2.0.0-preview.86" />
     <PackageReference Include="xunit" Version="2.5.1" />
     <PackageReference Include="xunit.runner.visualstudio" Version="2.5.1" />
     <PackageReference Include="coverlet.collector" Version="6.0.0" />
-    <PackageReference Include="Svg.Skia" />
+    <PackageReference Include="Svg.Skia" Version="$(SvgSkiaPackageVersion)" />
+    <PackageReference Update="Microsoft.CodeAnalysis.NetAnalyzers" Version="7.0.4" />
```

Comment:
oops?

Reply:
Is oops because of 
1) SkiaSharp with versions 
or 
2) `Update` instead of `Include` for `Microsoft.CodeAnalysis.NetAnalyzers`?

For 1) 

I tried to be consistent with versions `eng/Versions.props`. Is there a reason not to be?

https://github.com/dotnet/maui/blob/main/eng/Versions.props#L76-L77

```
    <_SkiaSharpVersion>2.88.6</_SkiaSharpVersion>
    <_SkiaSharpExtendedVersion>2.0.0-preview.86</_SkiaSharpExtendedVersion>
```

For 2)

`Include`started causing nuget (error NU1504) errors recently, so I switched to `Update`.

Error with `Include`:

```
2023-09-29T07:22:21.5358710Z /Users/builder/azdo/_work/2/s/src/SingleProject/Resizetizer/test/UnitTests/Resizetizer.UnitTests.csproj : error NU1504: Warning As Error: Duplicate 'PackageReference' items found. Remove the duplicate items or use the Update functionality to ensure a consistent restore behavior. The duplicate 'PackageReference' items are: Microsoft.CodeAnalysis.NetAnalyzers 7.0.4, Microsoft.CodeAnalysis.NetAnalyzers 7.0.4.
```